site2/docs/client-libraries.md

```diff
@@ -4,42 +4,74 @@ title: Pulsar client libraries
+Pulsar supports the following language specific client libraries:
 
-| Language  | Documentation                                                                 | Release note                                                             | Code repo                                                                     |
-|-----------|-------------------------------------------------------------------------------|--------------------------------------------------------------------------|-------------------------------------------------------------------------------|
-| Java      | - [User doc](client-libraries-java.md) <br/><br/> - [API doc](/api/client/)   | [Here](/release-notes/)                                                  | [Here](https://github.com/apache/pulsar/tree/master/pulsar-client)            |
-| C++       | - [User doc](client-libraries-cpp.md) <br/><br/> - [API doc](/api/cpp/)       | [Here](/release-notes/)                                                  | [Here](https://github.com/apache/pulsar/tree/master/pulsar-client-cpp)        |
-| Python    | - [User doc](client-libraries-python.md) <br/><br/> - [API doc](/api/python/) | [Here](/release-notes/)                                                  | [Here](https://github.com/apache/pulsar/tree/master/pulsar-client-cpp/python) |
-| WebSocket | [User doc](client-libraries-websocket.md)                                     | [Here](/release-notes/)                                                  | [Here](https://github.com/apache/pulsar/tree/master/pulsar-websocket)         |
-| Go client | [User doc](client-libraries-go.md)                                            | [Here](https://github.com/apache/pulsar-client-go/blob/master/CHANGELOG) | [Here](https://github.com/apache/pulsar-client-go)                            |
-| Node.js   | [User doc](client-libraries-node.md)                                          | [Here](https://github.com/apache/pulsar-client-node/releases)            | [Here](https://github.com/apache/pulsar-client-node)                          |
-| C#        | [User doc](client-libraries-dotnet.md)                                        | [Here](https://github.com/apache/pulsar-dotpulsar/blob/master/CHANGELOG) | [Here](https://github.com/apache/pulsar-dotpulsar)                            |
+| Language  | Documentation                          | Release note                                                                   | Code repo                                                                        |
+| --------- | -------------------------------------- | ------------------------------------------------------------------------------ | -------------------------------------------------------------------------------- |
+| Java      | [User doc](client-libraries-java.md)   | [Bundled](/release-notes/)                                                     | [Bundled](https://github.com/apache/pulsar/tree/master/pulsar-client)            |
```

Comment:
API docs can be found in user doc, we don't have to inline it with `<br/>` here.

Reply:
Suggest keeping API doc links in the table, because it makes the info more prominently since they are frequently used by developers.

Reply:
OK. Updated at 74ea3719a89e6a25bf82de7b3922be8fac7965b1.